ironfish/src/fileStores/config.ts

```diff
@@ -27,6 +27,11 @@ const MEGABYTES = 1000 * 1000
 
 export type ConfigOptions = {
   bootstrapNodes: string[]
+  /**
+   * STUN servers to use for inititating WebRTC connections.
+   * TODO: Use our own STUN servers
+   */
```

Comment:
I think we can delete this TODO here.

Reply:
The user who mentioned this to me said they would be more comfortable with a projects' STUN server. We can discuss offline though about user perception

Reply:
I would keep these todos to actual technical work that needs to be done rather than future product decisions. Like, this is more of a ticket in linear than a TODO.